docs/cli/argo_plugin.md

```diff
@@ -0,0 +1,50 @@
+## argo plugin
+
+manage plugins
+
+```
+argo plugin [flags]
```

Comment:
I think the answer is "no," but might there be any reason to nest the current functionality under `argo plugin executor`?

Reply:
hmm, maybe that might be a good idea